taxa_with_state.rq

```diff
@@ -8,6 +8,8 @@ PREFIX dc: <http://purl.org/dc/elements/1.1/>
 PREFIX StandardState: <http://purl.obolibrary.org/obo/CDAO_0000045>
 PREFIX bd: <http://www.bigdata.com/rdf/search#>
 # Replace the prefix value with the state URI of interest
+# These URIs are not persistent across KB updates; this example will return no 
+# results unless you provide a valid state URI.
```

Comment:
Is there a way to just randomly sample (e.g., take the first one) from a list of character state URIs? That's assuming that there is no such thing as a character state URI in the database for which there wouldn't be rows in the result.

Reply:
That's an interesting idea—it can probably be done.